be/src/seekGraph/middleware.ts

```diff
@@ -24,26 +24,34 @@ export const createSeekGraphMiddleware = async ({
   path,
   userAgent,
   seekApiUrlOverride,
-}: SeekGraphMiddlewareOptions): Promise<Middleware> => {
+}: SeekGraphMiddlewareOptions): Promise<Router.Middleware> => {
   const schema = await createSchema({
     getPartnerToken,
     userAgent,
     seekApiUrlOverride,
   });
 
   const server = new ApolloServer({
-    context: createContext,
+    cache: 'bounded',
+    csrfPrevention: true,
     introspection: debug,
-    debug,
     plugins: [
       debug
-        ? ApolloServerPluginLandingPageGraphQLPlayground()
+        ? ApolloServerPluginLandingPageLocalDefault()
         : ApolloServerPluginLandingPageDisabled(),
     ],
     schema,
   });
 
   await server.start();
 
-  return server.getMiddleware({ path });
+  const router = new Router().post(
+    path,
+    bodyParser(),
```

Comment:
Should we set `enableTypes: ['json']`:

https://www.npmjs.com/package/koa-bodyparser#options